### DISCUSSION TITLE
Projects

### DISCUSSION THREAD

**Discussion by logikaljay** [community/community#3165] [Mobile]
It would be great if mobile app users could create and manage projects on their smartphones/tablets.

**Comment by candyho** [comment]
Hey thanks for your feedback! Feel free to upvote on this topic related to Projects to help boost awareness of this request! - #3023
Also appreciate if you can share more details with us in the thread above 👆 - how would you expect to manage projects on mobile? Do you expect to view the full kanban board on mobile? What are the key project features that you think we should cater on mobile?
